docs/pool_connection.md

```diff
@@ -0,0 +1,246 @@
+ Object({"account_id": String("cosmos1kcwadpmfreuvdrvkz7v79ydclfnn4ukdhp57c2"), "alias": String("some_alias_1"), "pub_key": String("27taHHmKLcxZEHQPmiNHuXTXQYF7u2CxGrzKQMxBZALsQ")})
+ ]
+ ```
+
+#### **indy_cheqd_keys_sign**
+This method can sign a transaction by using a key which can be found by `alias`
+
+Input parameters:
+
+* `wallet_handle` - integer, which is connected to previously created and opened wallet.
+* `alias` - human-readable representation of alias to user, 
+* `tx_raw` - byte representation of transaction,
+* `tx_len` - length of string with bytes of transaction,
+
+As result, raw byte's string is expected.
+```
+[10, 146, 1, 10, 134, 1, 10, 37, 47, 99, 104, 101, 113, 100, 105, 100, 46, 99, 104, 101, 113, 100, 110, 111, 100, 101, 46, 99, 104, 101, 113, 100, 46, 77, 115, 103, 67, 114, 101, 97, 116, 101, 78, 121, 109, 18, 93, 10, 45, 99, 111, 115, 109, 111, 115, 49, 120, 51, 51, 120, 107, 106, 100, 51, 103, 113, 108, 102, 104, 122, 53, 108, 57, 104, 54, 48, 109, 53, 51, 112, 114, 50, 109, 100, 100, 52, 121, 51, 110, 99, 56, 54, 104, 48, 18, 10, 116, 101, 115, 116, 45, 97, 108, 105, 97, 115, 26, 11, 116, 101, 115, 116, 45, 118, 101, 114, 107, 101, 121, 34, 8, 116, 101, 115, 116, 45, 100, 105, 100, 42, 9, 116, 101, 115, 116, 45, 114, 111, 108, 101, 18, 4, 109, 101, 109, 111, 24, 192, 2, 18, 97, 10, 78, 10, 70, 10, 31, 47, 99, 111, 115, 109, 111, 115, 46, 99, 114, 121, 112, 116, 111, 46, 115, 101, 99, 112, 50, 53, 54, 107, 49, 46, 80, 117, 98, 75, 101, 121, 18, 35, 10, 33, 2, 59, 126, 95, 52, 102, 213, 99, 251, 102, 62, 148, 101, 72, 226, 188, 243, 222, 31, 35, 148, 19, 127, 79, 75, 79, 37, 160, 132, 193, 33, 148, 7, 18, 4, 10, 2, 8, 1, 18, 15, 10, 9, 10, 4, 99, 104, 101, 113, 18, 1, 48, 16, 224, 167, 18, 26, 64, 130, 229, 164, 76, 214, 244, 157, 39, 135, 11, 118, 223, 29, 196, 41, 92, 247, 126, 129, 194, 18, 154, 136, 165, 153, 76, 202, 85, 187, 195, 40, 69, 10, 206, 165, 238, 223, 245, 35, 140, 92, 123, 246, 110, 23, 39, 32, 215, 239, 230, 196, 146, 168, 5, 147, 9, 67, 113, 242, 163, 0, 223, 233, 73]
+```
+
+## Pool
+
+### List of methods
+* [`indy_cheqd_pool_add`](#indy_cheqd_pool_add)
+* [`indy_cheqd_pool_get_config`](#indy_cheqd_pool_get_config)
+* [`indy_cheqd_pool_get_all_config`](#indy_cheqd_pool_get_all_config)
+* [`indy_cheqd_pool_broadcast_tx_commit`](#indy_cheqd_pool_broadcast_tx_commit)
+* [`indy_cheqd_pool_abci_query`](#indy_cheqd_pool_abci_query)
+* [`indy_cheqd_pool_abci_info`](#indy_cheqd_pool_abci_info)
+
+#### **indy_cheqd_pool_add**
+This method is needed for adding information about pool which will be used to connect to.
+
+Input parameters:
+* `alias` - is a human-readable string,
+* `rpc_address` - address for connecting to the node, like `http://1.2.3.4:26657`, port `26657` is default value.
+* `chain_id` - identifier of the network.
```

Comment:
What is the purpose of this, if a node has already been added using cheqd Cosmos CLI?